### PR TITLE
Updated link to Binary by changing moa to rosa. Made a few other tweaks.

### PR DESCRIPTION
**Pull request by jonquilwilliams**
Peer and QE review needed. Please merge to 4.5 branch after QE verification.

Changed link to binary location from "moa" to "rosa". Please a few other minor tweaks to formatting/wording that do not affect technical accuracy.

JIRA Issue: https://issues.redhat.com/browse/OSD-5489

Manual Preview: http://file.rdu.redhat.com/jonwilli/moa/JIRA5489/getting_started_rosa/prereqs-rosa.html

@yufchang @vikram-redhat @jeremyeder  @okashi18

### CODE REVIEW

File: modules/rosa-installing-prerequisites.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ Complete the following prerequisites before creating your {product-title} cluste
 
 . Select the AWS account you want to use.
 +
-Unless you are just testing out {product-title}, it is recommended to use a dedicated AWS account to run any production clusters. If you are using AWS Organizations, you can use an AWS account within your organization or link:https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_create.html#orgs_manage_accounts_create-new[create a new one].
+Unless you are just testing out {product-title}, the best practice is to use a dedicated AWS account to run any production clusters. If you are using AWS Organizations, you can use an AWS account within your organization or link:https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_create.html#orgs_manage_accounts_create-new[create a new one].
```
Comment (bmcelvee) [inline modules/rosa-installing-prerequisites.adoc:15]: The ISG recommends not using "best practice". It has come up for me a few times, so I just wanted to mention it here. :)

File: modules/rosa-installing-prerequisites.adoc [hunk 2]
```diff
@@ -25,12 +25,12 @@ This user will have Programmatic access enabled and the AdministratorAccess poli
 +
 . Install and configure the AWS CLI.
 .. Follow the link:https://aws.amazon.com/cli/[AWS Command Line Interface] documentation to install and configure the AWS CLI for your operating system.
-.. Optional: Set a default AWS region. `rosa` evaluates regions in the following priority order:
+.. (Optional) Set a default AWS region. `rosa` evaluates regions in the following priority order:
```
Comment (bmcelvee) [inline modules/rosa-installing-prerequisites.adoc:28]: Another weird one for our recent style decisions is going with:

. Optional: <step>

**Comment by bmcelvee** [comment]
The link works and looks good! I would recommend reverting the style changes though.

**Comment by jonquilwilliams** [comment]
This PR was never merged for some reason even though the JIRA card was verified and closed. But, the binary changed again later on and was documented correctly by another card, so this PR is truly not needed and would render errors in the doc if merged now. So I am closing it accordingly.

**Pull request closed without merging**